Resources/public/js/views/actions/ez-translateactionview.js

```diff
@@ -0,0 +1,160 @@
+            }
+        };
+
+    /**
+     * Translate Action View
+     *
+     * @namespace eZ
+     * @class TranslateActionView
+     * @constructor
+     * @extends eZ.ButtonActionView
+     */
+    Y.eZ.TranslateActionView = Y.Base.create('translateActionView', Y.eZ.ButtonActionView, [Y.eZ.Expandable], {
+        initializer: function () {
+            this.events = Y.merge(this.events, events);
+            this.after({
+                'translateAction': this._toggleExpanded,
+                'expandedChange': this._setClickOutsideEventHandler,
+            });
+        },
+
+        /**
+         * Renders the action
+         *
+         * @method render
+         * @return Y.eZ.TranslateActionView the view itself
+         */
+        render: function () {
+            var container = this.get('container');
+
+            this._addButtonActionViewClassName();
+
+            container.setHTML(this.template({
+                actionId: this.get('actionId'),
+                disabled: this.get('disabled'),
+                label: this.get('label'),
+                hint: this.get('hint'),
+
+                location: this.get('location').toJSON(),
+                content: this.get('content').toJSON(),
+                translations: this.get('content').get('currentVersion').getTranslationsList()
+            }));
+
+            return this;
+        },
+
+        /**
+         * expandedChange event handler to define or detach the click outside
+         * event handler so that the view gets hidden when the user click
+         * somewhere else
+         *
+         * @method _setClickOutsideEventHandler
+         * @param {Object} e event facade of the expandedChange event
+         * @protected
+         */
+        _setClickOutsideEventHandler: function (e) {
+            if ( e.newVal ) {
+                this._clickOutsideSubscription = this.get('container').on(
+                    'clickoutside', Y.bind(this._hideView, this)
+                );
+            } else {
+                this._clickOutsideSubscription.detach();
+            }
+        },
+
+        /**
+         * Toggles the expanded state
+         *
+         * @method _toggleExpanded
+         * @protected
+         */
+        _toggleExpanded: function () {
+            this.set('expanded', !this.get('expanded'));
+
+            return this;
+        },
+
+        /**
+         * Hides the expanded view
+         *
+         * @method _hideView
+         * @protected
+         */
+        _hideView: function () {
+            this.set('expanded', false);
+        },
+
+        /**
+         * Returns the hint for translate button
+         *
+         * @return {String}
+         */
+        _getTranslateButtonHint: function () {
+            var translations = this.get('content').get('currentVersion').getTranslationsList(),
+                countAll = translations.length,
+                moreTranslations = Y.clone(translations);
```

Comment:
in such case, it's better to use `Y.merge(translations);` instead of `Y.clone` as we don't need a deep cloning.